requirements.txt

```diff
@@ -30,4 +30,7 @@ types-requests==2.32.0.20240907
 s3fs==2024.10.0
 pyarrow==18.1.0
 pyathena==3.9.0
-asana==3.2.3
\ No newline at end of file
+google-analytics-data==0.18.15
+google-auth-oauthlib==1.2.1
```

Comment:
do we need this oauth lib since we use service accounts? can you please test this after removing this library?

Reply:
Yeah, We don't need this. It's working fine after removing these library.